### PR TITLE
Add storage infos in web

### LINKED ISSUE #2158: Add summary info of storage in web
We need to add APIs for Storage Service to gather summaries and display them on web.

### PR DESCRIPTION
**Pull request by hekaisheng**
## What do these changes do?

Display storage infos in web

## Related issue number

Fixes #2158

### CODE REVIEW

File: mars/services/storage/core.py [hunk 1]
```diff
@@ -355,8 +357,12 @@ async def __post_create__(self):
         await self._create_storage_handler_actors()
         # create actor for transfer
         await self._create_transfer_actors()
+        # create task for uploading storage usages
+        self.ref().upload_storage_info.tell_delay(delay=1)
```
Comment (wjsi) [inline mars/services/storage/core.py:361]: Set the return value to `self._upload_task`.

File: mars/services/storage/core.py [hunk 2]
```diff
@@ -493,3 +499,19 @@ async def _setup_storage(self,
...
+            for band, level_to_quota in self._quotas.items():
+                for level, quota_ref in level_to_quota.items():
+                    total, used = await quota_ref.get_quota()
+                    storage_info = StorageInfo(storage_level=level,
+                                               total_size=total,
+                                               used_size=used)
+                    upload_tasks.append(
+                        self._cluster_api.set_band_storage_info.delay(band, storage_info))
+            await self._cluster_api.set_band_storage_info.batch(*upload_tasks)
+            self.ref().upload_storage_info.tell_delay(delay=1)
```
Comment (wjsi) [inline mars/services/storage/core.py:517]: Ditto.

File: mars/services/storage/core.py [hunk 3]
```diff
@@ -493,3 +499,19 @@ async def _setup_storage(self,
...
+            upload_tasks = []
+            for band, level_to_quota in self._quotas.items():
+                for level, quota_ref in level_to_quota.items():
+                    total, used = await quota_ref.get_quota()
+                    storage_info = StorageInfo(storage_level=level,
+                                               total_size=total,
+                                               used_size=used)
+                    upload_tasks.append(
+                        self._cluster_api.set_band_storage_info.delay(band, storage_info))
+            await self._cluster_api.set_band_storage_info.batch(*upload_tasks)
```
Comment (wjsi) [inline mars/services/storage/core.py:516]: Will this call block the actor?

**Review by wjsi — CHANGES_REQUESTED** [review]
`set_node_disk_info` not called. Pease also revert changes in `package-lock.json` and use `npm>=7.0.0`.

File: mars/services/storage/core.py [hunk 4]
```diff
@@ -493,3 +500,34 @@ async def _setup_storage(self,
...
+            self._upload_task = self.ref().upload_storage_info.tell_delay(delay=1)
+
+    async def upload_disk_info(self):
+        from ..cluster import DiskInfo
+
+        disk_infos = []
+        if self._cluster_api is not None:
+            if 'disk' in self._init_params:
```
Comment (wjsi) [inline mars/services/storage/core.py:528]: `and`

File: mars/services/storage/core.py [hunk 5]
```diff
@@ -493,3 +500,34 @@ async def _setup_storage(self,
...
+                    total, used = await quota_ref.get_quota()
+                    used = int(used)
+                    if total is not None:
+                        total = int(total)
+                    storage_info = StorageInfo(storage_level=level,
+                                               total_size=total,
+                                               used_size=used)
+                    upload_tasks.append(
+                        self._cluster_api.set_band_storage_info.delay(band, storage_info))
+            await self._cluster_api.set_band_storage_info.batch(*upload_tasks)
```
Comment (wjsi) [inline mars/services/storage/core.py:520]: Make upload task an independent while-true task started in `__post_create__`.

**Review by wjsi — CHANGES_REQUESTED** [review]

**Review by wjsi — APPROVED** [review]
LGTM

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**